trunk/Admin/js/build.js

```diff
@@ -0,0 +1,1001 @@
+let AdminQuickbarActions = {
```

Comment:
What's the purpose of this file?

Reply:
This is generated via gulp

Reply:
That doesn't answer the question. I'm assuming the `build.min.js` contains a minified version of this file. So what is the purpose of generating this file? Is it actually being used?

Reply:
No, it is not used, but deleting it manually would be pointless, as it would be regenerated with each Gulp run.
But that's how it comes from our WBBP and is the case in all of our plugins. If you wanted to change something, you would have to change that at the WBBP too.

Reply:
I have now removed it. But we have a look to WBBP in near future.